src/utils/tests/test-utils.js

```diff
@@ -1,17 +0,0 @@
-import React from 'react'
-import { render } from '@testing-library/react'
-import { ThemeProvider } from 'styled-components'
-import theme from '../theme'
-
-const AllTheProviders = ({ children }) => {
-  return <ThemeProvider theme={theme}>{children}</ThemeProvider>
-}
-
-const customRender = (ui, options) =>
-  render(ui, { wrapper: AllTheProviders, ...options })
-
-// re-export everything
```

Comment:
IMO an abstraction layer entirely hiding @testing-library/react behind a utils file is not worth it here vs using the library that many people know directly. Also not all tests will actually need this.